src/builder.ts

```diff
@@ -143,12 +143,21 @@ export interface Options {
    * @default false
    */
   experimentalCRUD?: boolean
+  /**
+   * Map of GraphQL scalar types to be used by the library for the Prisma scalars
+   *
+   * When not provided, the scalar types will be passthrough.
+   *
+   * @default {}
+   */
+  scalars?: Partial<Record<Typegen.GetGen<'scalars'>, GraphQLScalarType>>
   computedInputs?: GlobalComputedInputs
 }
 
 export interface InternalOptions extends Options {
   dmmf?: DmmfDocument // For testing
   nexusBuilder: Nexus.PluginBuilderLens
+  nexusPrismaImportId?: string,
```

Comment:
Same stuff as for `nexus`. If the lib is not hoisted, the import to `nexus-prisma/typegen` in the typegen file will fail, so we'll probably need to re-expose that on nexus and change the `importId` to something like `nexus-plugin-prisma/schema/typegen`